docs/source/notes/cuda.rst

```diff
@@ -54,6 +54,26 @@ Below you can find a small example showcasing this::
         f = torch.randn(2).cuda(cuda2)
         # d.device, e.device, and f.device are all device(type='cuda', index=2)
 
+.. _tf32_on_ampere:
+
+TensorFloat-32(TF32) on Ampere devices
+--------------------------------------
+
+Starting from `CUDA 11`_ and the `Ampere architecture`_, it is possible to compute
+matrix multiplications and convolutions for ``torch.float32`` data type using the
+new `TensorFloat-32`_ tensor cores. TensorFloat-32 is a new math mode that approaches
+up to 10x speedups by sacrificing precision. If you are using a device with TF32 support,
+the default math mode will be TF32. The precision for TF32 will be enough for most AI
+use cases, but if you are seeing precision issue, try manually turning it off:
+
+.. code:: python
+
+  torch.backends.cuda.matmul.use_tf32 = False  # for matrix multiplications
```

Comment:
Should we include a warning that the flag is global and should not be switch simultaneously by multiple threads?

Reply:
There is no warning for other flags, so no for this.